gluecodium/src/main/resources/templates/dart/DartProperty.mustache

```diff
@@ -18,4 +18,17 @@
   ! License-Filename: LICENSE
   !
   !}}
-{{!! TODO: APIGEN-1789: implement }}
\ No newline at end of file
+{{#set property=this}}
+{{#getter}}
+{{#unless comment.isEmpty}}{{prefix comment "/// "}}
+{{/unless}}
+{{#if isStatic}}static {{/if}}{{resolveName property.typeRef}} get {{resolveName visibility}}{{resolveName property}}{{!!
+}}{{#unless skipBody}}{{>dart/DartFunctionBody}}{{/unless}}{{#if skipBody}};{{/if}}
```

Comment:
Naming skipBody instead "declareOnly", "abstractProperty" or "interfaceProperty" would make the intention clearer.